packages/interfaces/src/filter.ts

```diff
@@ -20,7 +20,7 @@ export type SubscriptionCallback<T extends IDecodedMessage> = {
 export type SubscribeOptions = {
   keepAlive?: number;
   pingsBeforePeerRenewed?: number;
-  maxMissedMessagesThreshold?: number;
+  enableLightPushFilterCheck?: boolean;
```

Comment:
note that removing `maxMissedMessagesThreshold` is a breaking change

Reply:
it is but it wasn't used afaik

Reply:
if it's exposed externally we don't really know if consumers use it. But it's probably not a huge fix on their end if that's the case